doc/whats_new.rst

```diff
@@ -33,6 +33,7 @@ Bug
 - Fix problem in copying CTF files on Network File System due to a bug upstream in Python by `Mainak Jas`_ (`#174 <https://github.com/mne-tools/mne-bids/pull/174/files>`_)
 - Fix problem in copying BTi files. Now, a utility function ensures that all the related files
   such as config and headshape are copied correctly, by `Mainak Jas`_ (`#135 <https://github.com/mne-tools/mne-bids/pull/135>`_>)
+- Fix name of "sample" and "value" columns on events.tsv files, by `Ezequiel Mikulan`
```

Comment:
I suggest you add your name to the bottom of the file as a link. Also, you need to add an underscore to make it render correctly 
```
`Ezequiel Mikulan`_
```
Also, could you please add another link to this PR like in the other what's new entries?

Example: 
```
(`#185 <https://github.com/mne-tools/mne-bids/pull/185>`_)
```

Reply:
Thanks, sorry, I added the underscore and the PR. My name was already at the bottom because of a previous PR